<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>瓶子识别</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ico_src.qrc">
    <normaloff>:/mainWindow_ico/mainWindow_ico.png</normaloff>:/mainWindow_ico/mainWindow_ico.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(195, 227, 218)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="main_layout" stretch="6,2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout" stretch="0,0,0">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="main_account_but">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(217, 191, 119);
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>回收</string>
         </property>
         <property name="icon">
          <iconset resource="ico_src.qrc">
           <normaloff>:/mainWindow_ico/recycle_ico.png</normaloff>:/mainWindow_ico/recycle_ico.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>90</width>
           <height>90</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="main_identify_but">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(216, 235, 181);
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>种类</string>
         </property>
         <property name="icon">
          <iconset resource="ico_src.qrc">
           <normaloff>:/mainWindow_ico/kind_ico.png</normaloff>:/mainWindow_ico/kind_ico.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>90</width>
           <height>90</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="main_examine_but">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(99, 154, 103);
border-radius: 5px;</string>
         </property>
         <property name="text">
          <string>用户</string>
         </property>
         <property name="icon">
          <iconset resource="ico_src.qrc">
           <normaloff>:/mainWindow_ico/user_ico.png</normaloff>:/mainWindow_ico/user_ico.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>90</width>
           <height>90</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout_2" stretch="1,1,1">
       <property name="spacing">
        <number>50</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QPushButton" name="main_question_but">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>111111</width>
           <height>11111</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:5px;
background-color:rgb(33, 182, 178);</string>
         </property>
         <property name="text">
          <string>遇到问题</string>
         </property>
         <property name="icon">
          <iconset resource="ico_src.qrc">
           <normaloff>:/mainWindow_ico/question_icon.png</normaloff>:/mainWindow_ico/question_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mian_convert_but">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:5px;
background-color:rgb(33, 182, 178);</string>
         </property>
         <property name="text">
          <string>积分兑换</string>
         </property>
         <property name="icon">
          <iconset resource="ico_src.qrc">
           <normaloff>:/mainWindow_ico/convert_icon.png</normaloff>:/mainWindow_ico/convert_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="main_suggestion_but">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius:5px;
background-color:rgb(33, 182, 178);</string>
         </property>
         <property name="text">
          <string>意见和建议</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icon/idea_icon.png</normaloff>icon/idea_icon.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ico_src.qrc"/>
 </resources>
 <connections/>
</ui>
